echoregions/formats/regions2d.py

```diff
@@ -90,6 +91,16 @@ def offset(self, val):
         """Set the depth offset to apply to y values"""
         self._offset = float(val)
 
+    @property
+    def plotter(self):
+        if self._plotter is None:
+            if not self.data:
+                raise ValueError(
+                    "Input file has not been parsed; call `parse_file` to parse."
+                )
+            self._plotter = Region2DPlotter(self)
```

Comment:
Is this done here so that `Region2DPlotter`  does not get created unless the person needs to plot? Trying to understand why this initialization is not in `__init__`.